### PR TITLE
Compress and upload artifact to github actions.

### PR DESCRIPTION
**Pull request by xuzhao9**
This is the first step to make TorchBench v0 score available in [Edwards's dashboard](https://ezyang.github.io/pytorch-ci-hud/build2/pytorch-master)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -18,3 +18,12 @@ jobs:
       - name: Run benchmark
         run: |
           bash ./.github/scripts/run-nightly-nodocker.sh
+      - name: Compress artifact
+        run: |
+          export TODAY=$(date "+%Y%m%d")
```
Comment (wconstab) [inline .github/workflows/main.yml:23]: is it better to include hour/min/sec to this, in case some day this job runs twice for some reason?

**Comment by wconstab** [comment]
woah cool idea! what are the next steps? i am happy to hear about that!

**Review by wconstab — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (xuzhao9) [reply .github/workflows/main.yml:23]: Good catch!

Comment (xuzhao9) [reply .github/workflows/main.yml:23]: Good point!

**Pull request merged**